src/index.ts

```diff
@@ -145,13 +150,39 @@ export default class ServerlessConventions {
     // Check that the service name does not contain the word "service"
     if (serviceName.toLowerCase().includes('service')) {
       errors.push(
-        `Warning: Service name should not include the word "service"`
+        `Warning: Service name "${serviceName}" should not include the word "service"`
       );
     }
 
     // Check the length of the service name is not greater than x
     if (serviceName.length > 23) {
```

Comment:
23 feels like a magic number.. not sure exactly what to do about it but this is a smell for sure

Reply:
This was done by me a while ago. I agree it's not great. 23 is the max "safe" length for a service